modules/nf-core/metabat2/jgisummarizebamcontigdepths/main.nf

```diff
@@ -0,0 +1,38 @@
+process METABAT2_JGISUMMARIZEBAMCONTIGDEPTHS {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::metabat2=2.15" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/metabat2:2.15--h986a166_1' :
+        'quay.io/biocontainers/metabat2:2.15--h986a166_1' }"
+
+    input:
+    tuple val(meta), path(bam), path(bai)
+
+    output:
+    tuple val(meta), path("*.txt.gz"), emit: depth
+    path "versions.yml"                    , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    export OMP_NUM_THREADS=$task.cpus
```

Comment:
again i know this is probs fine because its on nf-core but what does this do? I don't see where it's called or used?

Reply:
In the subworkflow `mapping_depth.nf` - it feels hidden, maybe I should add some comments in the `main.nf` workflow at the mapping and depth step what is actually being done in this subworkflow